server/src/main/java/org/cloudfoundry/identity/uaa/authentication/manager/VmidentityAuthenticationManager.java

```diff
@@ -137,7 +134,7 @@ public Authentication authenticate(Authentication req) throws AuthenticationExce
                         upn,
                         upn,
                         upn,
-                        userId.getDomain().equalsIgnoreCase(systemDomain) ? OriginKeys.UAA : OriginKeys.LDAP,
```

Comment:
for future, it would be really nice if we could separate a logical change vs reformat/var rename change.
it is really hard to review when this is all munged together. thanks!